deployments/keycloak-security/README.md

```diff
@@ -0,0 +1,79 @@
+# Local Minio Deployment
+
+## Description
+This is a local sample deployment that protects an ion-ingest, an ion-store, and an ion-search service behind a
+a gateway service. The gateway service uses a keycloak instance for an oauth IDP, and minio as
+Amazon S3 compatible storage for the ingest service.
```

Comment:
❓ Is the gateway's job to startup the ION apps as well? Or should the gateway just be an app that can be run alongside whatever ION apps you want to run in your own `docker-compose` file?

Reply:
The second one, gateway's routes can be hooked up to any services you specify in it's config. This compose file is just an example of how it would be connected to CDR's apps.